Description: Install AuErrorDB to /usr/share/libaudio2 and update code to look there.

Author: Steve McIntyre <93sam@debian.org>
Bug-Debian: http://bugs.debian.org/637326
--- nas-1.9.3.orig/lib/audio/Imakefile
+++ nas-1.9.3/lib/audio/Imakefile
@@ -79,7 +79,7 @@ XCOMM JET - see if we should avoid strea
     COMPAT_OBJS = AuGetBest.o
 #endif /* ProjectX < 5 */
 
-     EDB_DEFINES = -DERRORDB=\"$(LIBDIR)/AuErrorDB\"
+     EDB_DEFINES = -DERRORDB=\"/usr/share/libaudio2/AuErrorDB\"
 
         DEFINES = $(MALLOC_DEFINES) $(SYSV_DEFINES)
    CONN_DEFINES = ConnectionFlags
@@ -137,11 +137,11 @@ SpecialObjectRule(globals.o,/**/,$(CONN_
 BuildIncludes($(HEADERS),audio,..)
 InstallMultiple($(HEADERS),$(INCDIR))
 #if ProjectX >= 5
-InstallNonExecFile(AuErrorDB,$(LIBDIR))
+InstallNonExecFile(AuErrorDB,/usr/share/libaudio2)
 #else
 install::
 	$(MKDIRHIER) $(LIBDIR)
-InstallNonExec(AuErrorDB,$(LIBDIR))
+InstallNonExec(AuErrorDB,/usr/share/libaudio2)
 #endif
 
 #if defined(HPArchitecture) && !defined(LinuxArchitecture) && !defined(GNUArchitecture) && !defined(DarwinArchitecture)
@@ -151,7 +151,7 @@ XAULIB = -lXau
 #endif
 
 #if defined(LinuxArchitecture) || defined(GNUArchitecture) || defined(SunArchitecture) || defined(__FreeBSD_kernel__) || defined(DarwinArchitecture)
-REQUIREDLIBS = -L$(USRLIBDIR) -lXt $(XAULIB) -lm 
+REQUIREDLIBS = -L$(USRLIBDIR) -lXt $(XAULIB) -lm  $(LDFLAGS_DEF)
 #endif
 
 #if defined(cygwinArchitecture)
